spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/support/GitSkipSslValidationCredentialsProvider.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2013-2018 the original author or authors.
```

Comment:
I think you just need 2018

Reply:
Thanks @pivotal-dylan-roberts - fixed now. Also sorted out the whitespacing issues you pointed out. Cheers!